Sources/EdgeNetworkHandlers/EdgeEventError.swift

```diff
@@ -12,18 +12,35 @@
 
 import Foundation
 
-/// Error or warning information for a sent EdgeRequest
-struct EdgeEventError: Codable {
+/// Error information for a sent EdgeRequest
+struct EdgeEventError: Codable, Equatable {
+    /// Error message
+    let title: String?
+
+    /// Detailed message of the error
+    let detail: String?
+
+    /// Error code info
+    let status: Int?
+
+    /// Namespaced error code
+    let type: String?
 
-    /// Encodes the event to which this error/warning is attached as the index in the events array in EdgeRequest
+    /// Encodes the event to which this error is attached as the index in the events array in EdgeRequest
     let eventIndex: Int?
 
-    /// Error message
-    let message: String?
+    /// A report for the error containing additional information
+    let report: EdgeErrorReport?
+}
 
-    /// Error code info
-    let code: String?
+// MARK: - EdgeErrorReport
+struct EdgeErrorReport: Codable, Equatable {
+    // An array of errors represented as strings
+    let errors: [String]?
+
+    /// Request ID corresponding to the error
+    let requestId: String?
```

Comment:
We already include the requestId we have on the client, not sure if it is required to decode it from here.

Reply:
I'd rather match the incoming schema as best as we can rather than omit properties, even if not used.